lldb/source/Symbol/SwiftASTContext.cpp

```diff
@@ -2345,6 +2345,18 @@ llvm::Triple SwiftASTContext::GetTriple() const {
 ///
 /// TODO: Make Swift more robust.
 llvm::Triple SwiftASTContext::GetSwiftFriendlyTriple(llvm::Triple triple) {
+  // Set the vendor to `unknown` on Windows as the Swift standard library is
+  // overly aggressive in matching the triple.  The vendor field is
+  // initialized to `pc` by LLDB though there is no official vendor associated
```

Comment:
is lldb initializing to `pc` a bug that needs to be fixed?

Reply:
I think that it should default to `LLVM_DEFAULT_TARGET_TRIPLE` (which I do actually set to `x86_64-unknown-windows-msvc`).  The thing is, normally this wouldn't be an issue.  However, the Swift module loading is extra sensitive about the match being 100% identical.  However, because `pc` is the default hardcoded vendor, this breaks down.